### PR TITLE
Users/shbhawsi/dontusevstsnugetpush

### PR DESCRIPTION
**Pull request by shubham90**

### CODE REVIEW

File: Tasks/Common/utility-common/package-lock.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
       "resolved": "https://registry.npmjs.org/argparse/-/argparse-1.0.10.tgz",
       "integrity": "sha512-o5Roy6tNG4SL/FOkCAN6RzjiakZS25RLYFrcMttJqbdd8BWrnA+fGz57iN5Pb06pvBGvl5gQ0B48dJlslXvoTg==",
```
Comment (jotaylo) [inline Tasks/Common/utility-common/package-lock.json]: revert the changes to this file

Comment (shubham90) [reply Tasks/Common/utility-common/package-lock.json]: done

Comment (shubham90) [reply Tasks/Common/utility-common/package-lock.json]: done
